test/docker/compose.go

```diff
@@ -431,12 +434,30 @@ func (d *Compose) Start(ctx context.Context) (*DockerCompose, error) {
 		envSettings["CLUSTER_GOSSIP_BIND_PORT"] = "7102"
 		envSettings["CLUSTER_DATA_BIND_PORT"] = "7103"
 		envSettings["CLUSTER_JOIN"] = fmt.Sprintf("%s:7100", Weaviate)
+		envSettings["RAFT_PORT"] = "8302"
+		envSettings["RAFT_INTERNAL_PORT"] = "8303"
+		envSettings["RAFT_JOIN"] = "node1:8300"
 		container, err := startWeaviate(ctx, d.enableModules, d.defaultVectorizerModule,
 			envSettings, networkName, image, hostname)
 		if err != nil {
 			return nil, errors.Wrapf(err, "start %s", hostname)
 		}
 		containers = append(containers, container)
+
+		hostname = WeaviateNode3
```

Comment:
The behaviour of the change is that when starting a cluster, 2 more additional nodes will start and join as voter the main node. In the end we will have a 3 node voting cluster for acceptance test, or a single vote node if we are not running in cluster mode.